### ISSUE TITLE
Issue creating Theme on Windows Server 2012 R2

### ISSUE THREAD

**Issue by Anseur01** [Jaredk3nt/VSThemeGenerator#5]
I just installed this extension after trying out VS code, coming from notepad++, and as a result, I wanted to try to reproduce some of my appearance settings from that.

After installing this extension and following the instructions on usage from the homepage of the extension, when running the Generate Theme command, I get the following error

![error](https://user-images.githubusercontent.com/59394129/73119886-d79c2900-3f5f-11ea-8b12-eac17fd99421.PNG)

I was first using the 'user' instillation of VS code, so I tried installing the system version, with the same result. I tried running it as an admin, with the same result.

My settings.json (I think that is where I'm supposed to put this?) looks like this:
`{
    "workbench.startupEditor": "newUntitledFile",
    "workbench.colorTheme": "Trail Theme",
    "Find replace": true,
    "Save active file": true,
    "themeGenerator.colors": {
        "background": "#050505",
        "foreground": "#090909",
        "primary": "#327193",
        "keywords": "#43baba",
        "functions": "#F0F275",
        "strings": "#b9ecfa",
        "numbers": "#8789c0",
        "punctuation": "#494f53",
        "operators": "#327193",
        "comment": "#007809",
        "tags": "#43baba",
        "attributes": "#3fa5c1",
        "properties": "#43baba",
        "builtins": "#b5ca8d",
        "variables": "#4C5FFF",
        "types": "#3fa5c1"
    }
}`
About VS code:
Version: 1.41.1 (system setup)
Date: 2019-12-18T14:58:56.166Z
Electron: 6.1.5
Chrome: 76.0.3809.146
Node.js: 12.4.0
V8: 7.6.303.31-electron.0
OS: Windows_NT x64 6.3.9600

Edit: I tried putting the color info into the workspace.json of the default workspace instead like so:
`{
	"folders": [
		{
			"path": "E:\\inetpub\\wwwroot\\resources"
		}
	],
	"themeGenerator.colors": {
		"background": "#050505",
		"foreground": "#090909",
		"primary": "#327193",
		"keywords": "#43baba",
		"functions": "#F0F275",
		"strings": "#b9ecfa",
		"numbers": "#8789c0",
		"punctuation": "#494f53",
		"operators": "#327193",
		"comment": "#007809",
		"tags": "#43baba",
		"attributes": "#3fa5c1",
		"properties": "#43baba",
		"builtins": "#b5ca8d",
		"variables": "#4C5FFF",
		"types": "#3fa5c1"
	}
}
`
But this did not help.

Edit2: I just ran the system version as an admin and I noticed this error when running it as well this time:

`
Failed to properly create necessary file structure. VSThemeGenerator Requires undefined/.vscode/extensions/generatedTheme/themes to exist.`

**Comment by Jaredk3nt** [comment]
Yeah I wasn't super careful with pathing for the files on this so it looks like windows will have some problems. I will try to update it to use proper path utils so it will support both mac and windows.
